ReSharper.FSharp/test/src/FSharp.Tests.Host/FSharpTestHost.fs

```diff
@@ -41,12 +44,26 @@ type FSharpTestHost
         |> Option.map (fun options ->
             options.OtherOptions
             |> Array.choose (fun o -> if o.StartsWith("-r:") then Some (o.Substring("-r:".Length)) else None)
-            |> Array.map (fun s -> FileSystemPath.TryParse(s))
+            |> Array.map (FileSystemPath.TryParse)
             |> Array.filter (fun p -> not p.IsEmpty && directory.IsPrefixOf(p))
             |> Array.map (fun p -> p.Name)
             |> List)
         |> Option.defaultWith (fun _ -> List())
 
+    let setFeatures (rdFeatures: RdSetFeatures) =
+        for feature in rdFeatures.Features do
+            match experimentalFeatures.TryGetValue feature with
+            | null -> ()
+            | value -> value.Dispose()
+
+            if rdFeatures.Enable then
+                experimentalFeatures.[feature] <-
+                    match feature with
+                    | RdExperimentalFeatures.AllowFormatter -> FSharpRegistryUtil.AllowFormatterCookie.Create()
```

Comment:
Could it be done in a more generic way, please? If we're to use it for other features, it'll be better not to duplicate it here for each one.